v2/src/plugins/markdownVariables.json

```diff
@@ -31,7 +31,8 @@
         "codeImportRecipeName": "session",
         "goModelName": "sessmodels",
         "goModelNameForInit": "&sessmodels",
-        "recipeNameCapitalLetters": "Session"
+        "recipeNameCapitalLetters": "Session",
+        "recipePreBuiltUINameCapitalLetters": "SessionPreBuiltUI"
```

Comment:
there is no pre built UI for session right?

Reply:
There may be some later, e.g., the access denied screen. We can remove this, for now, I'm just noting this.